presto-docs/src/main/sphinx/functions/lambda.rst

```diff
@@ -0,0 +1,67 @@
+============================
+Lambdas and Lambda Functions
+============================
+
+Lambda Expression
+-----------------
+
+Lambda expressions in Presto are written with ``->``. For example::
```

Comment:
On all of these, you can remove the "For example:" and just do the colon at the end of the previous sentence. See JSON functions for example.